src/main/java/org/gitlab4j/api/ProjectApi.java

```diff
@@ -2176,4 +2177,23 @@ public Project unstarProject(Object projectIdOrPath) throws GitLabApiException {
         Response response = post(expectedStatus, (Form) null, "projects", getProjectIdOrPath(projectIdOrPath), "unstar");
         return (response.readEntity(Project.class));
     }
+
```

Comment:
No need to introduce a new type for this, please re-implement `getProjectLanguage(Integer projectId)` as follows:
```
    /**
     * Get languages used in a project with percentage value.
     *
     * Get /projects/:id/languages
     *
     * @param projectId the ID of the project
     * @return a Map instance with the language as the key and the percentage as the value
     * @throws GitLabApiException if any exception occurs
     * @since GitLab 10.8
     */
    public Map<String, Float> getProjectLanguages(Integer projectId) throws GitLabApiException {

        if (projectId == null) {
            throw new RuntimeException("projectId cannot be null");
        }

        Response response = get(Response.Status.OK, null, "projects", projectId, "languages");
        return (response.readEntity(new GenericType<Map<String, Float>>() { }));
    }
```